src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	/**
+	 * @return the feedbackToUser
```

Comment:
Better to write method description for simple method than using `@return`. Refer to #188